app/src/main/cpp/skyline/loader/nsp.cpp

```diff
@@ -62,4 +64,18 @@ namespace skyline::loader {
         icon->Read(buffer);
         return buffer;
     }
+
+    void NspLoader::ExtractTickets(const std::shared_ptr<vfs::Directory> &dir, const std::shared_ptr<crypto::KeyStore> &keyStore) {
```

Comment:
You can make this a static function in the c++ file itself and remove it from the header which might be cleaner